client/js/controllers/user_list_controller.js

```diff
@@ -12,6 +12,8 @@ const EmptyView = require('../views/empty_view.js');
 
 class UserListController {
     constructor(ctx) {
+        this._pageController = new PageController();
```

Comment:
Why was it moved here?

Reply:
If an error is thrown before the page controller is initialized it doesn't show up on the page and only in the console.